WindowsClient/_Data/_Actions/GlobalEnvironmentVariables.cs

```diff
@@ -1,22 +1,12 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace WindowsClient._Data._Actions
 {
     public static class GlobalEnvironmentVariables
     {
-        public static bool Set(string variable, string value)
+        public static void Set(string variable, string h1, float latitude, float longitude)
```

Comment:
here, is needly other argument to object height, sorry for this, is a my mistake in this task definitions.